pkgs/os-specific/linux/kernel/hardened/patches.json

```diff
@@ -1,4 +1,14 @@
 {
+    "4.14": {
```

Comment:
4.14-hardened is back

Reply:
Fix is in progress: https://github.com/NixOS/nixpkgs/pull/277708
So this is hopefully the last round where we need to watch out for that :)